wallet/src/de/schildbach/wallet/ui/send/SendCoinsActivityViewModel.kt

```diff
@@ -37,7 +37,6 @@ open class SendCoinsActivityViewModel(application: Application) : AndroidViewMod
     }
 
     val walletApplication = application as WalletApplication
-    val wallet = walletApplication.wallet!!
```

Comment:
This line was causing a crash if the wallet was `null`.  `wallet` is not used by this view model.

The fact that `wallet` is `null` is a big problem, so removing the crash here will probably mean that another crash will appear elsewhere.